app/javascript/admin/controllers/sidebar_controller.js

```diff
@@ -0,0 +1,35 @@
+import { Controller } from 'stimulus';
+
+// eslint-disable-next-line no-restricted-syntax
+export default class SidebarController extends Controller {
+  static targets = [
+    'submenu'
+  ];
+
+  disableCurrentNavItem() {
+    const activeMenuId = this.submenuTargets.filter((item) => item.classList.contains("show"))[0].id
+    const activeButton = document.getElementById(`${activeMenuId}_button`);
+    activeButton.setAttribute("disabled", true)
```

Comment:
Sorry I know this is merged already, but since the overall work is still in progress I thought I'd ask - why do we disable this? 

Disabling it means it's hidden to screen reader users, so they no longer know that we're in an expanded sub-menu. I'm also wondering why we want to stop users from clicking again to collapse it 🤔

Reply:
ooh that's a good point. This was part of the [interaction design specifications](https://github.com/forem/forem/pull/12967#issuecomment-796895169) from @lisasy, perhaps there's alternatives we can discuss as an extension of the RFC.